src/test/java/com/bobocode/dialect/PersistenceUnitPropertiesTest.java

```diff
@@ -0,0 +1,58 @@
+package com.bobocode.dialect;
+
+import com.bobocode.bibernate.configuration.PersistenceUnitProperties;
+import com.bobocode.parser.YamlPropertyParser;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.Map;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PersistenceUnitPropertiesTest {
+    private static final String H_2 = "H2";
```

Comment:
**(nit)** I would rename it to H2. like SA

Reply:
Ok.